src/components/Card.jsx

```diff
@@ -9,7 +9,17 @@ import no_img from "../image/card-no-image.svg";
 // CardHeader -> ~ago 표시되는 부분(card.createdAt 활용)
 // CardContent -> 설명 표시되는 부분(card.description 활용)
 // CardFooter -> 생성날짜 표시되는 부분(card.createdAt 활용)
-function Card({ id, createdAt, url, title, description, imageSource }) {
+function Card({
+  id,
+  createdAt,
+  url,
+  title,
+  description,
+  imageSource,
+  created_at,
+  image_source,
+  showStarButton,
+}) {
```

Comment:
P2:
반복되는 속성은 Card component를 사용하는 곳에서 formatting 한후 필요한 값만 전달하면 좋겠습니다.

```suggestion
/**
* const CardList = () => {
* const data = useGetData('/api/cardList')
* const formattedCardList = formatCardData(data);
*
* return formattedCardList.map((cardProps) => <Card {...cardProps}/>)
* }
*/
function Card({
  id,
  createdAt,
  url,
  title,
  description,
  imageSource,
  showStarButton,
}) {
```